x-pack/test/accessibility/apps/uptime.ts

```diff
@@ -61,7 +60,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     it('overview alert popover controls', async () => {
       await uptimeService.overview.openAlertsPopover();
+      await toasts.dismissAllToasts();
```

Comment:
Whenever you see an error like https://github.com/elastic/kibana/issues/90555 it's probably a toasts issue, even if you're actively opening/closing popovers/flyouts.